### PR TITLE
Set release version

### PR DESCRIPTION
**Pull request by ali-ince**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-49.surge.sh

Updated pages:
- [Installation](https://neo4j-docs-spark-49.surge.sh/spark/5/installation)

File: modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -8,21 +8,26 @@ Make sure to match both the Spark version and the Scala version of your setup.
...
-|`neo4j-connector-apache-spark_2.12`
-|`4.1.5_for_spark_3`
-|`{exact-connector-version}_for_spark_3`
-
-|*Scala 2.13*
-|`org.neo4j`
-|`neo4j-connector-apache-spark_2.13`
-|`4.1.5_for_spark_3`
-|`{exact-connector-version}_for_spark_3`
+| Spark Version | Artifact ID (*Scala 2.12*) | Artifact ID (*Scala 2.13*)
```
Comment (nvitucci) [inline modules/ROOT/pages/installation.adoc:13]: I'm fine with the restructuring of the table, but I have a doubt about calling the whole package name "Artifact ID". Is there a better substitute for the `GroupID:ArtifactID:Version` combination?

Comment (fbiville) [reply modules/ROOT/pages/installation.adoc:13]: maybe let's just call it `Artifact`?

Comment (ali-ince) [reply modules/ROOT/pages/installation.adoc:13]: Yeah, happy with `Artifact` 👍

Comment (nvitucci) [reply modules/ROOT/pages/installation.adoc:13]: It might do. Or maybe `Artifact name`?

Comment (ali-ince) [reply modules/ROOT/pages/installation.adoc]: I mean theoretically we could use `Maven coordinate` as well?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-spark-49.surge.sh

Updated pages:
- [Installation](https://neo4j-docs-spark-49.surge.sh/spark/5/installation)

**Review by nvitucci — APPROVED** [review]
Sounds good then.

File: modules/ROOT/pages/installation.adoc (hunk 1 above)
Comment (nvitucci) [reply modules/ROOT/pages/installation.adoc]: Yes, although it may be overly specific. I think we can go with `Artifact`.

**Pull request merged**